docs/cimpl6Authoring.md

```diff
@@ -98,69 +96,80 @@ Decide if you need to import any namespaces (e.g. OBF).
 
-Map the model specific properties to either FHIR resource/profile elements, or OBF.
+Map the model specific `Property` to either FHIR **resource/profile element**s, or OBF.
 
-[CIMPL Language Reference Guide](cimpl6LanguageReference.md)
+[CIMPL Language Reference Manual](cimpl6LanguageReference.md#map-file)
 
 ## Value Set File
 
-Create this file when you need to constrain an `Element Value` of `concept` in the Class file and there is not an existing value set to reference.  
+Create a ValueSet file when you need to constrain an `Element` with data type `concept` in the Class file, and there is not an existing value set to reference.  
 
-To avoid redundant value sets, try to find an existing value set that fits your use case.
+To avoid redundant value sets, try to find an existing value set that fits your use case rather than creating a custom, IG specific value set.
 
-[CIMPL Language Reference Guide](cimpl6LanguageReference.md)
+[CIMPL Language Reference Manual](cimpl6LanguageReference.md#value-set-file)
 
 ## Content Profile File
 
-Create a *Content Profile* file if there are properties in your model that are critical to IG compliance and _must_ be supported. Any inherited model properties that are not already defined as [MustSupport](https://www.hl7.org/fhir/conformance-rules.html) are listed in the same file.
+Create a Content Profile file if there are properties in your model that are critical to IG compliance and _must_ be supported. Any inherited model properties that are not already defined as [**MustSupport**](https://www.hl7.org/fhir/conformance-rules.html) are listed in the same file.
+
+A list of classes _not_ to be profiled, designated with the [No-Profile](#specifying-no-profile-elements) `NP` tag.
 
-[CIMPL Tooling Reference Guide](cimpl6ToolingReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#content-profile-file)
 
 ## Front Matter
 
-Introductory text and possibly detailed text is required for IG readers and implementers to understand the background and intended use of your IG.
+Introductory text is required, and detailed text is optional for IG readers and implementers to understand the background and intended use of your IG.
 
 Create this information and declare the location in the Configuration file.
 
 Reviewing a [published IG](http://www.fhir.org/guides/registry/) might help you structure this material.
 
-[CIMPL Tooling Reference Guide](cimpl6ToolingReference.md)
+[CIMPL Tooling Reference Manual](cimpl6ToolingReference.md#front-matter-files)
 
 ## FHIR Examples
 
 A FHIR IG can be generated without examples, however it is recommended to include examples.
 
-The FHIR IG generation process requires that an examples folder/directory be defined in the Configuration file. The folder may be empty.
+The examples file location is declared in the Configuration file.
```

Comment:
The location of the examples is declared in the Configuration File.

Reply:
Changed sentence as recommended.

Reply:
Not quite as I wrote it. The location can be a file or folder. I purposely avoided the word "file"